api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -20,6 +24,8 @@
 
 
     protected Logger log = LoggerFactory.getLogger(getClass());
+    private PersonService personService = Context.getPersonService();
+    public static final String LOCATION_ATTRIBUTE_TYPE_NAME = "accessControlLocation";
```

Comment:
What has our code got to do with the attribute type name? Do you really need it?

Reply:
Then How can we track the person attribute in the future implementations? Actually, I thought to move this to a separate constants file for extending the usage.

Reply:
What do you mean by future implementations?
Is there any implementation which cannot just use a uuid value for the global property to specify which person attribute type stores the location?

Reply:
Why would you want to enforce the name? Do you want the Chinese to also use English names?

Reply:
I got errors when tried to save the attributeType without name. So How can I overcome from this issue?

Reply:
Why do you even save one? The user will have configured it by the time they registered patients.

Reply:
I'm sorry, If I missed to understand your concept 
Actually, I'm talking about the PersonAttributeType. It should be created once for this reason.  Then the uses can create the person attributes using that personAttributeType when the register the patients. 
That's why I need the personAttributeType Name here

Reply:
The end user creates it and points you to it using a global property. So you do not need to create it.

Reply:
So you mean, I need to get both name and Uuid from the Global property. Right?

Reply:
What do you need the name for?

Reply:
Oopss, I just got your point. Let the user create the person Attribute type and point that by a global property.
I will update my code soon.

Reply:
Let see my code, I have totally removed the personAtrribute creation during the startup. Now it will get the personAttribute through UUID from the global property